app/locales/en/translations.ts

```diff
@@ -1702,6 +1702,12 @@ export default {
             expandSideNav: 'Expand registration form navigation',
             collapseSideNav: 'Collapse registration form navigation',
         },
+        'registries-top-nav': {
+            showRegistrationNavigation: 'Show registration navigation',
```

Comment:
Should this be `Show registration form navigation` to be consistent with the other translations? or change the other translations to match this one?